DEVELOP.md

```diff
@@ -9,65 +9,66 @@ Dependencies:
 4. [RabbitMQ](http://www.rabbitmq.com/) for workers
 
 
-### Setting up CycleDash
+### 1. Install Dependencies
+
+```bash
+brew install Python                #brew will install the latest Python 2.7.9, which includes pip
+brew install postgres				
+brew install rabbitmq				
+pip install virtualenv
+```
+
+
+### 2. Set up CycleDash
 
 ```bash
 virtualenv venv                    # Initialize a new virtual environment.
-source venv/bin/activate           # Activate your virtual environment.
+source venv/bin/activate           # Activate your virtual environment. This is the only command you will need to run every time you start the app.
 pip install -r requirements.txt    # Install python requirements into virtualenv
-npm install                        # Install JS requirements
+npm install                        # Install JS requirements and packages in package.json
```

Comment:
The "JS requirements" are "packages", so how about just

`# Install JS requirements from package.json`